console-plugins/dense-mapping-constraints-plugin/src/dense-mapping-alignment.cc

```diff
@@ -81,8 +81,19 @@ bool computeAlignmentForCandidatePairsImpl<resources::PointCloud>(
 
   // Initialize the aligner if we haven't already.
   if (!*aligner_ptr) {
-    *aligner_ptr =
-        regbox::BaseController::make(regbox::Aligner::LpmIcp, "ADMC Aligner");
+    if (FLAGS_dm_candidate_alignment_type == "PclIcp") {
+      *aligner_ptr =
+          regbox::BaseController::make(regbox::Aligner::PclIcp, "ADMC Aligner");
+    } else if (FLAGS_dm_candidate_alignment_type == "PclGIcp") {
+      *aligner_ptr = regbox::BaseController::make(
+          regbox::Aligner::PclGIcp, "ADMC Aligner");
+    } else if (FLAGS_dm_candidate_alignment_type == "LpmIcp") {
+      *aligner_ptr =
+          regbox::BaseController::make(regbox::Aligner::LpmIcp, "ADMC Aligner");
+    } else {
+      LOG(FATAL) << "Selected alignment for dense mapping constraints is not "
```

Comment:
Should we maybe turn this into a `LOG(ERROR)` and select a default type instead of crashing the system?